src/main/java/nl/hsac/fitnesse/util/iban/IbanGenerator.java

```diff
@@ -239,4 +247,15 @@ protected String getControlNumber(String bankCode, String account, String countr
 
         return controlNr;
     }
+
+    protected static String[] readLines(String resourceFile) {
+        URL resource = IbanGenerator.class.getResource("/iban/" + resourceFile);
+        File codesFile = new File(resource.getFile());
+        try {
+            List<String> allCodes = Files.readAllLines(codesFile.toPath(), UTF8);
+            return allCodes.toArray(new String[allCodes.size()]);
+        } catch (IOException e) {
+            throw new IllegalStateException(e);
+        }
+    }
```

Comment:
This will slow down your tests, so I don't like it. Unless it is in the jar.
It does of course enable you to add and remove bank codes if you want to, provided the txt is in the test setup and not packaged in the jar. If not, I do not really see a benefit other than two files instead of one. Enlighten me? :-)

Reply:
It doesn't really slow down your tests as the file is only read at startup, and not for each generation. It is indeed packaged in the jar, which still allows you to change it by providing another file 'earlier' in the classpath, but that is not the real benefit I see.
In my opinion it makes it easier to see the functionality (the Java code) separate from the data used (the other file), AND the data is easier to maintain as no quotes, commas and indentation has to be maintained.